app/views/messages/index.html.erb

```diff
@@ -4,43 +4,46 @@
 <main class="messaging-container" ng-app="Communicant">
   <h3 class="message-title">Messages</h3>
 
+
+  <form class="add-message" ng-controller="NewMessageController" ng-submit="submit()">
+    <!-- <textarea class="add-message-body" ng-model="newMessage.body" placeholder="Type your message here..."></textarea> -->
+    <select class="select-premessage" ng-model="newMessage.message.body" name="select">
+      <option value="">Select an option</option>
+      <option>Check your mail</option>
+      <option>An event has been changed in your list of events</option>
+      <option>An expense has been changed in the list of expenses</option>
+      <option>A Change in Information panel</option>
+    </select>
+    <!-- {{newMessage.body}} -->
+    <button class="message-send" type="submit" name="submit-message">Send</button>
+  </form>
+
   <section class="chatbox">
-    <!-- <div class="chatbox-scroll"> -->
 
-     <ul class="messaging-list" id="scroll-bottom" ng-controller="ListOfMessagesController">
-    <!-- <% @messages.each do |message| %>
-      <li class="message">
-        <!-- <div class="message-name"><%# message.name %></div> -->
-        <!-- <div class="message-date"><%# message.date %></div>
-        <div class="message-time"><%# message.time %></div>
-        <div class="message-body"><%# message.body %></div>
-      </li> -->
-    <!-- <% end %> -->
+
+
+
+     <ul class="messaging-list" ng-controller="ListOfMessagesController">
+
+       <i class="fa fa-refresh fa-spin" ng-show="loading"></i>
```

Comment:
This tag isn't valid as a direct child of a `<ul>` tag, so the browser is probably "helping" by moving it. Unfortunately, if you put it outside the `<ul>`, the controller doesn't apply anymore, and if inside the repeating `<li>` it gets repeated along with everything else. Other options?